Dockerfile

```diff
@@ -14,25 +14,28 @@ RUN apk add --no-cache \
       openldap-dev \
       postgresql-dev
 
+FROM builder AS netbox_builder
 WORKDIR /install
 
-RUN pip install --prefix="/install" --no-warn-script-location \
-# gunicorn is used for launching netbox
-      gunicorn \
-      greenlet \
-      eventlet \
-# napalm is used for gathering information from network devices
-      napalm \
-# ruamel is used in startup_scripts
-      'ruamel.yaml>=0.15,<0.16' \
-# django_auth_ldap is required for ldap
-      django_auth_ldap \
-# django-storages was introduced in 2.7 and is optional
-      django-storages
-
 ARG NETBOX_PATH
-COPY ${NETBOX_PATH}/requirements.txt /
-RUN pip install --prefix="/install" --no-warn-script-location -r /requirements.txt
+COPY ${NETBOX_PATH}/requirements.txt requirements-container.txt /
```

Comment:
Well, it's not that bad I've just learned. Because django-* libs will already install django and take a long time. It doesn't really matter therefore.